css/mediaQueries.css

```diff
@@ -0,0 +1,16 @@
+@media screen and (max-width: 870px){
```

Comment:
you can use em unit in media query vikram sir told in lecture

Reply:
yes i can , but to avoid calculations, i have used absolute units.